### ISSUE TITLE
Expected indentation of spaces

### ISSUE THREAD

**Issue by movie4** [stylelint/stylelint#1847]
Hello!

How to remove 'Expected indentation spaces" warning?

See code and warning below

Thanks

``` css
.catalog-item-popup {
  & .popup-content {
    width: 740px;
    background: #fff;
    border: solid 1px #808080;
    box-shadow: 0 10px 40px 4px rgba(0, 0, 0, 0.3);
  }
  &-content {
    width: 100%;
    margin: 15px 0 0 0;
  }
  &__inner {
    width: 100%;
    display: table;
  }
    &__left {
      display: table-cell;
      vertical-align: middle;
    }
```

.stylelintrc
```
{
  "extends": "stylelint-config-standard",
  "rules": {
    "function-url-quotes" : "single",
    "string-quotes": "single",
    "indentation": {
      "hierarchicalSelectors": true
    },
    "rule-non-nested-empty-line-before": "always-multi-line",
    "at-rule-empty-line-before": "never",
    "shorthand-property-no-redundant-values": "never"
  }
}
```
![screen shot 2016-08-27 at 13 57 38](https://cloud.githubusercontent.com/assets/1005998/18026385/66c6161e-6c5e-11e6-89ac-3d4c5bc1af36.png)

**Comment by jeddy3** [comment]
@movie4 Use `null` to turn off the indentation rule i.e. `indentation: null`. See: [configuration guide](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#rules).

The `hierarchicalSelectors` option was removed in `7.0.0`. See: [release planning](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/release-planning.md). 

If you wish to use that style of indentation please consider creating a plugin for the community (e.g. `stylelint-hierarchical-selectors-indentation`). See: [plugin development guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md).

**Comment by movie4** [comment]
@jeddy3 
Thanks for quick response!
